mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -263,8 +267,8 @@ func InvokeGenerationFromSheet(wg *sync.WaitGroup) error {
 	return nil
 }
 
-// For registrants eg: meshery, whose components needs to be directly created by referencing the sheet.
-// the sourceURL contains the "rangeString" : the value that describes the sheet in which the components are listed, and their starting row to ending row.
+// For registrants eg: meshery, whose components needs to be directly created by referencing meshery/schemas repo.
+// the sourceURL contains the path of models component definitions
 func GenerateDefsForCoreRegistrant(model utils.ModelCSV) error {
 	totalComps := 0
 	version := "1.0.0"
```

Comment:
This hardcoding is no good. This needs to be source from tagged releases of `meshery/schemas`. 

Why is the generation process for `meshery-core` models being treated any differently from other models; why is there a function specifically written for generation of Meshery Core/Shape/Flow models? @MUzairS15

Reply:
This hardcoding of version was a concern when we were referencing the Comps from the sheet (I somewhere mentioned it elsewhere I believe), because the version was not captured in the sheet.

But now because they are sourced from the schemas repo and live inside the versioned directory we can remove this.

I realised this yesterday and thought I will convey @Yashsharma1911 but later forgot.

Reply:
@Yashsharma1911 make sense?

Reply:
Let's remove this hardcoding and update generator according to new changes in `meshery/schema` in iterative PR